include/libopencm3/stm32/l0/memorymap.h

```diff
@@ -23,6 +23,7 @@
 /* --- STM32 specific peripheral definitions ------------------------------- */
 
 /* Memory map for all busses */
+#define FLASH_BASE			(0x08000000U)
```

Comment:
Right, nothing actually uses this. It just was there for F0 and F1, and some F1 code uses it.